<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stockWindow</class>
 <widget class="QDialog" name="stockWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Stocks</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="imageLabel">
     <property name="minimumSize">
      <size>
       <width>369</width>
       <height>200</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:img/market1.jpeg);</string>
     </property>
     <property name="pixmap">
      <pixmap>:/images/livreur_cartons.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="formLayout">
     <item>
      <widget class="QWidget" name="formWidget" native="true">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
  border-radius: 8px;
  border: 1px solid #e0e4e7;
  padding: 5px 15px;
}

QLineEdit:focus {
  border: 1px solid #d0e3ff;
}

QLineEdit::placeholder {
  color: #767e89;
}

QComboBox {
  border-radius: 8px;
  border: 1px solid #e0e4e7;
  padding: 5px 15px;
}

QComboBox:focus {
  border: 1px solid #d0e3ff;
}</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Approvisionner le restaurant</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="nomDeLIngrDientLabel">
          <property name="text">
           <string>Nom de l'ingrédient</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="nomDeLIngrDientLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="dishNameLabel">
          <property name="text">
           <string>Recette</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="dishNameLineEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="typeLabel">
          <property name="text">
           <string>Quantité</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="quantitLineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="commandButton">
       <property name="styleSheet">
        <string>
         QPushButton {
             background-color: #4CAF50;
             color: white;
             border-radius: 10px;
             padding: 10px;
         }
         QPushButton:hover {
             background-color: #45a049;
         }
        </string>
       </property>
       <property name="text">
        <string>Commander</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
